patches/@nrk+core-datepicker+4.0.0.patch

```diff
@@ -111,29 +111,17 @@ index f1b5c26..2232247 100644
 +
  var version = "4.0.0";
  
-+/**
-+* getUUID
-+* @return {String} A generated unique ID
-+*/
-+
-+function getUUID() {
-+  return Date.now().toString(36) + Math.random().toString(36).slice(2, 5)
-+}
-+
  /**
- * closest
- * @param {Element} el Element to traverse up from
-@@ -669,7 +732,8 @@ var closest = function () {
+@@ -669,7 +723,7 @@ var closest = function () {
  
  function customElementToReact(elementClass, options) {
    if (options === void 0) options = {};
 -  var name = elementClass.name || String(elementClass).match(/function ([^(]+)/)[1]; // String match for IE11
-+  var suppliedName = options.name || elementClass.name || String(elementClass).match(/function ([^(]+)/)[1] // String match for IE11
-+  var name = suppliedName + "-" + getUUID();
++  var name = options.name || elementClass.name || String(elementClass).match(/function ([^(]+)/)[1] // String match for IE11
```

Comment:
Hvis du fjerner disse må de nesten erstattes med en annen metode hvis vi vil unngå den potensielle bug'en som gjorde dette nødvendig i første omgang:

På grunn av minifisering får _alle_ NRK-komponentene samme navn i DOMet når de brukes i produksjon. Siden de får versjonsnummeret som suffix _kan_ det likevel fungere, men hvis du f.eks. bruker både core-toggle og core-datepicker med samme versjon på samme side vil minst en av dem tryne siden begge rendres ut som f.eks. `<t-3-0-9>` 😑

Reply:
Er kanskje bare `core-datepicker` som er igjen etter denne PRen når jeg tenker meg om. I så fall er det ikke like farlig, men kan fortsatt være et issue hvis klientene bruker NRK-komponenter selv.

Reply:
Oooh, så det er ikke problemet med flere av _den samme_ komponenten, det er når det er to (med samme versjonsnummer) minifisert på samme side. Hmm. Husker du om dere prøvde å ha komponentnavnet som et pre/postfix her, i stedet for UUIDen? Eller blir det bare minifisert vekk det også?

Reply:
Tror greia er at komponentnavnet blir hentet fra navnet på klassen/funksjonen, som jo blir minifisert i prod 😑 Men Det skal jo i utgansgpunktet gå an å sende inn et navn i options ja. Kan hende jeg sørget for å gjøre det i en annen del av patchen, men husker ikke helt.. Hvis man får gjort det skal det jo i prinsippet funke fint!